### ISSUE TITLE
[E2] Clean code nombre de las stats de las unidades

### ISSUE THREAD

**Issue by vicentethomas** [franciscogazitua/IIC2113-2024#40]
Hola! En el libro se menciona explícitamente que HP no es un buen nombre para una variable:
>We should avoid words whose entrenched meanings vary from our intended meaning. For example, hp, aix, and sco would be poor variable names because they are the names of Unix platforms or variants.

Mi duda es que más adelante se menciona que el nombre de las variables depende mucho del contexto y en casos donde el contexto sea claro hay más libertad. Entonces, para el caso de Fire Emblem, dejar las variables de stats con sus nombres HP, Atk, Spd, Def, Res sería incumplir clean code?

**Comment by RodrigoToroIcarte** [comment]
No, esos nombres están bien porque son nombres que vienen directo del dominio del problema. De hecho cualquier de nosotros que vea un atributo nombrado HP (en cualquiera de los proyectos) pensaría que se refiere a la vida de la unidad. Esa es una buena señal de que es un buen nombre. Lo que sería un mal nombre sería llamar HP a cualquier otra cosa que no se refiera al HP de la unidad :)
